### PR TITLE
Improve the output on install and batch imports commands

### PR DESCRIPTION
**Pull request by iulyanp**
**Description (for Contributor and Core Developer)**
Hello,

See https://github.com/akeneo/pim-community-dev/issues/5209

Here is the PR I promised on the commands. 
It improves the output on install and batch imports commands by printing the warnings and what is processing in the console if you use `-v` flag.

I also want to contribute to the `pim:product-asset:mass-upload` command. I noticed the command reads the images from `%kernel.root_dir%/file_storage/tmp/mass_upload_imported/{user}` but only from the first level of the folder. I think it would be helpful if it could read also from sub directories.
Since I can't create a PR to the EE bundle I created a gist file for this https://gist.github.com/iulyanp/6cc062c9b83d1c813a1800fd97a8be5f.

Please let me know what you think.

Best regards

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: src/Akeneo/Bundle/BatchBundle/Command/BatchCommand.php [hunk 1]
```diff
@@ -163,11 +164,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
         $job->getJobRepository()->updateJobExecution($jobExecution);
 
+        $verbose = $input->getOption('verbose');
         if (ExitStatus::COMPLETED === $jobExecution->getExitStatus()->getExitCode()) {
             $nbWarnings = 0;
             /** @var StepExecution $stepExecution */
             foreach ($jobExecution->getStepExecutions() as $stepExecution) {
                 $nbWarnings += count($stepExecution->getWarnings());
+                foreach ($stepExecution->getWarnings() as $warning) {
+                    if ($verbose) {
```
Comment (skeleton) [inline src/Akeneo/Bundle/BatchBundle/Command/BatchCommand.php]: put this line before `foreach` to avoid to foreach warnings if verbose option is false

**Comment by iulyanp** [comment]
Hello @skeleton,

Thank you for the feedback, hope you're doing well.
I fixed the verbose check.
What do you think about pim:product-asset:mass-upload command contribution?

Best regards

**Comment by fabienlem** [comment]
Hello @iulyanp!

Everything is OK on our side, a last thing you can do before we can merge your PR is to squash your commits. Thank you!

A pim:product-asset:mass-upload command contribution could be great, but technically speaking we cannot provide you an access to our enterprise repository.

Kind regards,
Fabien

**Comment by iulyanp** [comment]
Hello @fabienlem 

Sorry for my late response, I squashed my commits as you requested.

As for the `pim:product-asset:mass-upload` command contribution I already created a gist file with those changes. You have the link in my initial comment.

Best regards,

**Comment by pierallard** [comment]
Thanks for the contribution @iulyanp !!

**Pull request merged**